2/art/art.py

```diff
@@ -23,6 +24,32 @@ def art(A, b, iterations):
             if sqrd_norm <= 0:
                 continue
             else:
-                x += (b[j][0] - np.dot(row, x)) * np.conjugate(row) / sqrd_norm
+                x += relax * ((b[j][0] - np.dot(row, x)) * np.conjugate(row) / sqrd_norm)
```

Comment:
What is this relaxation factor?

Reply:
I was just experimenting with the `lambda` from the wikipedia article: https://en.wikipedia.org/wiki/Algebraic_reconstruction_technique